src/couch/src/couch_db_engine.erl

```diff
@@ -42,6 +42,12 @@
 -type purge_info() :: [{docid(), revs()}].
 -type epochs() :: [{Node::atom(), UpdateSeq::non_neg_integer()}].
 -type size_info() :: [{Name::atom(), Size::non_neg_integer()}].
+-type partition_info() :: [
+    {Partition::atom(), Partition::binary()} |
```

Comment:
odd. why aren't we saying `{partition, Partition::binary()}` here? using the same name twice is confusing and probably doesn't even work?